### PR TITLE
add new feature 

### PR DESCRIPTION
**Pull request by jedsada-gh**

### CODE REVIEW

File: app/src/main/java/com/jedsada/dangerplayground/Calculator.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.jedsada.dangerplayground
+
+class Calculator {
+    fun plus( x : Int, y : Int)=x+y
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/Calculator.kt:4]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/Calculator.kt#L4"></span>Unexpected spacing before “:”</td>
    </tr>
  </tbody>
</table>

Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/Calculator.kt:4]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/Calculator.kt#L4"></span>Missing spacing around “=”</td>
    </tr>
  </tbody>
</table>

Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/Calculator.kt:4]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/Calculator.kt#L4"></span>Missing spacing around “+”</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserModel.kt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package com.jedsada.dangerplayground
 
-data class UserDao (val username : String? = null,
+data class UserModel (val username : String? = null,
```
Comment (T-Jedsada) [inline app/src/main/java/com/jedsada/dangerplayground/UserModel.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/91c47854f91b127a896e7ea792d87395e683df47/app/src/main/java/com/jedsada/dangerplayground/UserModel.kt#L3"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>

Comment (T-Jedsada) [inline app/src/main/java/com/jedsada/dangerplayground/UserModel.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/91c47854f91b127a896e7ea792d87395e683df47/app/src/main/java/com/jedsada/dangerplayground/UserModel.kt#L3"></span>Unexpected spacing before “:”</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserData.kt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.jedsada.dangerplayground
+
+data class UserData(val username : String?=null,
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L3"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>

Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L3"></span>Unexpected spacing before “:”</td>
    </tr>
  </tbody>
</table>

Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L3"></span>Missing spacing around “=”</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserData.kt [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+            val email: String? = null,
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:4]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L4"></span>Unexpected indentation (expected 4, actual 12)</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserData.kt [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+            val gender: Boolean? = true,
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:5]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L5"></span>Unexpected indentation (expected 4, actual 12)</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserData.kt [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+            val age: Int? = 20,
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:6]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L6"></span>Unexpected indentation (expected 4, actual 12)</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UserData.kt [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+            val address: String? = null
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UserData.kt:7]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L7"></span>Unexpected indentation (expected 4, actual 12)</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+package com.jedsada.dangerplayground
+
+import android.widget.*
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt#L3"></span>Wildcard import</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+import org.w3c.dom.Text
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt:4]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt#L4"></span>Unused import</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+
+fun ImageView.url () {
+    // TODO: use glide for image loader
+}
+
+fun TextView.setColorByType (type : Int) {
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt:10]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt#L10"></span>Unexpected spacing before “:”</td>
    </tr>
  </tbody>
</table>

File: app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (jedsada-gh) [inline app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt:11]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/23ef695b7edee6269957dc477e5f4e3d434faaf5/app/src/main/java/com/jedsada/dangerplayground/UIExtensions.kt#L11"></span>Unexpected blank line(s) before “}”</td>
    </tr>
  </tbody>
</table>

**Pull request merged**